src/sync.ts

```diff
@@ -51,10 +51,11 @@ import { MatrixError, Method } from "./http-api";
 import { ISavedSync } from "./store";
 import { EventType } from "./@types/event";
 import { IPushRules } from "./@types/PushRules";
-import { RoomStateEvent } from "./models/room-state";
+import { RoomState, RoomStateEvent } from "./models/room-state";
 import { RoomMemberEvent } from "./models/room-member";
 import { BeaconEvent } from "./models/beacon";
 import { IEventsResponse } from "./@types/requests";
+import { IMarkerFoundOptions } from "./models/room-state";
```

Comment:
Can these not be in the existing import statement? (Sonar smell)

Reply:
Yes! 👍 I wish there was a lint for this